src/Security/Handler/SecurityHandlerInterface.php

```diff
@@ -23,22 +23,18 @@ interface SecurityHandlerInterface
     /**
      * @param string|array $attributes
      * @param mixed|null   $object
-     *
-     * @return bool
      */
-    public function isGranted(AdminInterface $admin, $attributes, $object = null);
+    public function isGranted(AdminInterface $admin, $attributes, $object = null): bool;
 
     /**
      * Get a sprintf template to get the role.
-     *
-     * @return string
      */
-    public function getBaseRole(AdminInterface $admin);
+    public function getBaseRole(AdminInterface $admin): string;
 
     /**
      * @return array<string, mixed>
      */
-    public function buildSecurityInformation(AdminInterface $admin);
+    public function buildSecurityInformation(AdminInterface $admin): array;
 
     /**
      * Create object security, fe. make the current user owner of the object.
```

Comment:
createObjectSecurity(AdminInterface $admin, object $object): void;
deleteObjectSecurity(AdminInterface $admin, object $object): void;

Same for implementations.